test/specs/validate-address.spec.js

```diff
@@ -391,7 +391,7 @@ describe("validateAddress()", () => {
     expect(isValid).to.be.a("boolean").and.to.equal(false);
 
     // The normalized address is null
-    expect(normalizedAddress).to.equal(undefined);
+    expect(normalizedAddress).to.equal(null);
```

Comment:
Couldn't tell if this was intentional based on the comment but it falls in line with the undefined/null/empty comment above.

Reply:
If the API doesn't return a normalized address, then the JavaScript SDK returns `undefined`, which is more idiomatic in JavaScript than `null`.  It's fine for the API to return `null`, since that's idiomatic JSON though.

Reply:
Does the SDK currently handle converting from something like `{normalizedAddress: null, isValid: true}` to `{isValid: true}` ensure the original version of this test would pass?  If not, does this need to be added? The API will always return a `normalizedAddress` key as part of the result.

Reply:
The SDK converts `{normalizedAddress: null, isValid: true}` from the API to `{normalizedAddress: undefined, isValid: true}`, so the original version of this test should pass.

Reply:
Gotcha. It doesn't pass with `undefined` so I'm not sure what's going on with the translation in the SDK. Do you want me to dig in or have someone else take a look?

Reply:
You're right.  I'm guessing this is probably because my implementation didn't return the `normalizedAddress` key at all if it was empty.  No biggie though.  Just change [this line in the SDK](https://github.com/ShipEngine/shipengine-js/blob/3c8e4e22a39690fe46c99f15227c450b5ce75b33/src/address/validate-address.ts#L59) to this:

```javascript
normalizedAddress: (normalizedAddress || undefined) && {
```